parsers/s01-parse/cyrille37/sympa_ww-logs.yaml

```diff
@@ -0,0 +1,75 @@
+#
```

Comment:
The comment can be moved into a `.md` file with the same naming as the `.yaml` file which will be shown on hub.crowdsec.net